docs/tutorial/windows-store-guide.md

```diff
@@ -5,7 +5,7 @@ Windows Platform. The new `.appx` format does not only enable a number of new
 powerful APIs like Cortana or Push Notifications, but through the Windows Store,
 also simplifies installation and updating.
 
-Microsoft [developed a tool that compiles Electron apps as `.appx` packages](http://github.com/catalystcode/electron-windows-store),
+Microsoft [developed a tool that compiles Electron apps as `.appx` packages](electron-windows-store),
```

Comment:
I think `electron-windows-store` needs to be wrapped in `()` instead of `[]`. Similar for other links below.